templates/navigation/links--oe-multilingual-content-language-block.html.twig

```diff
@@ -7,9 +7,8 @@
  */
 #}
 {% include '@oe_theme/compositions/ec-component-content-language-switcher/content-language-switcher.html.twig' with {
-  'expandable_id': expandable_id,
-  'unavailable': unavailable,
-  'current': current,
-  'options': options,
-  'icon_path': ecl_icon_path,
-} only %}
+  'expandable': expandable,
```

Comment:
I've confused a bit. Should we pass `expandable` or `expandable_id` variable?

Reply:
True, that's a mistake.